### ISSUE TITLE
Issue when inserting inline code after inclusion, in conjuction with --aux-directory

### ISSUE THREAD

**Issue by stdedos** [gpoore/minted#150]
Same code as issue #149, only I added before the next chapter:
```
	\begin{itemize}
		\item \sout{Lines [**-**]: Something?}
		X\mint{c}{code[y] = -1 * ecoFUN(code[x] / (tol * tol)) * (tol * tol); // tol = 17.4;}X
	\end{itemize}
```

minted package duplicates the previous included code (from file).

**Comment by gpoore** [comment]
I can't reproduce the issue. I'd suggest making sure that you're not using `frozencache`, and possibly deleting and recreating the cache.

If this is still happening after doing that and trying the fix for #149, let me know.

**Comment by stdedos** [comment]
I verify that 981fb70 also fixes this issue
